expenses-add-on/README.md

```diff
@@ -0,0 +1,58 @@
+# Create expense reports directly from email receipts
+
+_Allow your employees to directly import expense report information from their email._
+
+Last updated: July, 2019
+
+## Description
+
+Simplify how your employees do expense reporting with a Gmail Add-on and Google Sheets. Anytime a
+user receives an email receipt, they can open the add-on which automatically contains relevant 
+information about the expense. Users can edit that information and then submit the form to log 
+their expense into a spreadsheet.
+
+![demo](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/expenses-add-on/expenseit.gif)
+
+## Technology highlights
+
+- Use the [Card Service][cardservice] for Gmail add-ons to build a UI to works both on web and
+mobile. 
+- Store user values using Google Apps Script's [Properties Service][propertiesservice].
+
+[cardservice]: https://developers.google.com/apps-script/reference/card-service/
+[propertiesservice]:https://developers.google.com/apps-script/reference/properties/
+
+## Try it
+
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
+1. Select **Publish** > **Deploy from manifest**.
+1. Next to the entry **Latest Version (Head)** click **Get ID**.
+1. Select and copy the Deployment ID value.
+1. Close the dialog and the Deployments window.
+1. Open the [Gmail add-on settings][settings] tab.
+1. In the Add-ons tab, ensure that you have selected the Enable developer add-ons for my account checkbox.
+1. Paste your add-on's deployment ID into the Developer add-ons textbox and click Install.
```

Comment:
Button text should be formatted like menus and other labels. Here and a few lines below.